File: Ad-Hoc/pagina30/3307/3307.js
const fs  = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split("\n").map(Number)
const num_casos = input.shift()

const calcular_raio = (area) => Math.sqrt(area / (4 * 3.14))

for(let i = 0; i < num_casos; i++){
    let area = input[i]
    let raio = calcular_raio(area)

    if (raio < 12){
        let valor = area * 0.09
        console.log(`vermelho = R$ ${valor.toFixed(2)}`)
    }
    
    else if (raio >= 12 && raio <= 15){
        let valor = area * 0.07
        console.log(`azul = R$ ${valor.toFixed(2)}`)
    }
    
    else if (raio > 15){
        let valor = area * 0.05
        console.log(`amarelo = R$ ${valor.toFixed(2)}`)
    }
}

File: Ad-Hoc/pagina30/3307/3307.py
import math
def calcular_raio(area):
    raio = math.sqrt(area / (4 * 3.14))
    return raio
    
num_esferas = int(input())
for _ in range(num_esferas):
    area = int(input())
    raio = calcular_raio(area)
    
    if raio < 12: # vermelho R$0.09
        valor = area * 0.09
        print(f"vermelho = R$ {valor:.2f}")
     
    elif 12 <= raio <= 15: # azul R$0.07
        valor = area * 0.07
        print(f"azul = R$ {valor:.2f}")
       
    elif raio > 15: # amarelo R$0.05
        valor = area * 0.05
        print(f"amarelo = R$ {valor:.2f}")